test/models/post_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
```

Comment:
delete this file
you dont need do test

Reply:
![Screenshot from 2021-08-30 20-19-04](https://user-images.githubusercontent.com/87750131/131337822-802f004c-6958-4e7b-9ebc-df2fe3d6a3ff.png)